packages/flutter/lib/src/painting/continuous_rectangle_border.dart

```diff
@@ -3,58 +3,276 @@
 // found in the LICENSE file.
 
 import 'dart:math' as math;
+import 'dart:ui' as ui;
 
 import 'basic_types.dart';
-import 'border_radius.dart';
 import 'borders.dart';
 import 'edge_insets.dart';
 
-/// A rectangular border with smooth continuous transitions between the straight
-/// sides and the rounded corners.
+/// A rectangle border with continuous corners.
+///
+/// A shape similar to a rounded rectangle, but with a smoother transition from
+/// the sides to the rounded corners.
+///
+/// The rendered shape roughly approximates that of a superellipse. In this
```

Comment:
This is clearer, thanks

Reply:
Shall we add a link to some article (e.g., https://medium.muz.li/optical-effects-9fca82b4cd9a#efbc) to explain why developers may sometimes prefer this than rrect?